src/experimental/test/property/arbitraries/stable_b_tree_map_arb.ts

```diff
@@ -27,76 +25,31 @@ export function StableBTreeMapArb(
         .tuple(
             CandidValueAndMetaArb(context),
-            argumentInfoArb(),
             CandidValueAndMetaArb(context),
-            argumentInfoArb(),
             UniqueIdentifierArb('globalNames'),
             createUniquePrimitiveArb(
                 fc.nat({
                     max: 253
                 })
             )
         )
-        .map(
-            ([
-                keySample,
-                keySerializableType,
-                valueSample,
-                valueSerializableType,
+        .map(([keySample, valueSample, name, memoryId]) => {
+            const imports = new Set([
+                ...keySample.src.imports,
+                ...valueSample.src.imports,
+                'stableJson',
+                'StableBTreeMap'
+            ]);
+
+            const key = keySample.src.typeAnnotation;
+            const value = valueSample.src.typeAnnotation;
+
+            return {
                 name,
-                memoryId
-            ]) => {
-                const imports = new Set([
-                    ...keySample.src.imports,
-                    ...valueSample.src.imports,
-                    'stableJson',
-                    'StableBTreeMap'
-                ]);
-
-                const serializableArguments = getSerializableArguments(
-                    keySample,
-                    valueSample,
-                    keySerializableType,
-                    valueSerializableType,
-                    context.api
-                );
-
-                const key = keySample.src.typeAnnotation;
-                const value = valueSample.src.typeAnnotation;
-
-                return {
-                    name,
-                    imports,
-                    definition: `let ${name} = ${context.api === 'class' ? 'new' : ''} StableBTreeMap<${key}, ${value}>(${memoryId}${serializableArguments});`,
-                    keySample,
-                    valueSample
-                };
-            }
-        );
-}
-
-function getSerializableArguments(
-    keySample: StableBTreeMap['keySample'],
-    valueSample: StableBTreeMap['valueSample'],
-    keySerializableType: SerializableType,
-    valueSerializableType: SerializableType,
-    api: Api
-): string {
-    const keyArgument =
-        keySerializableType === 'STABLE_JSON' || api === 'class'
-            ? 'stableJson'
-            : keySample.src.typeObject;
-    const valueArgument =
-        valueSerializableType === 'STABLE_JSON' || api === 'class'
-            ? 'stableJson'
-            : valueSample.src.typeObject;
-
-    return `, ${keyArgument}, ${valueArgument}`;
-}
-
-function argumentInfoArb(): fc.Arbitrary<SerializableType> {
-    return fc.oneof(
-        fc.constant<'STABLE_JSON'>('STABLE_JSON'),
-        fc.constant<'CANDID_TYPE_OBJECT'>('CANDID_TYPE_OBJECT')
-    );
+                imports,
+                definition: `let ${name} = new StableBTreeMap<${key}, ${value}>(${memoryId}, stableJson, stableJson);`,
```

Comment:
Let's just delete the `stableJson` parameters, those are the default.

Reply:
done